### ISSUE TITLE
"Riddicule" on the last line

### ISSUE THREAD

**Issue by kevinlinxc** [cutenode/1x.engineer#156]
There's a typo on the last line riddicule -> ridicule

**Comment by kevinlinxc** [comment]
Ok the typo isn't in the source code for some reason

**Comment by kdambekalns** [comment]
Yeah, has been fixed with 71821acf1a1efad296dc459384c2d81d0428e88a - but the site shows an older state… My guess: https://1x.engineer/ is served from `master`, but `main` is the current state.

**Comment by majsty** [comment]
Is this the entire joke? 🙂
